dlf/plugins/pageview/class.tx_dlf_fulltext_eid.php

```diff
@@ -53,7 +53,7 @@ public function main($content = '', $conf = array ()) {
 
 		$url = \TYPO3\CMS\Core\Utility\GeneralUtility::_GP('url');
 
-		$fulltextData = file_get_contents($url);
+		$fulltextData = file_get_contents(\TYPO3\CMS\Core\Utility\GeneralUtility::getUrl($url);
```

Comment:
This change does not looks correct if `file_get_contents` call should be replaced.